tests/integ_tests/metrics_tests.rs

```diff
@@ -88,6 +91,17 @@ async fn prometheus_metrics_exported() {
     ));
     // Verify counter names are appropriate (don't end w/ '_total')
     assert!(body.contains("temporal_request{"));
+    // Verify non-temporal metrics meter does not prefix
+    let mm = rt.telemetry().get_metric_meter().unwrap();
+    let g = mm.inner.gauge(MetricParameters::from("mygauge"));
+    g.record(
+        42,
+        &MetricAttributes::OTel {
+            kvs: Arc::new(vec![]),
+        },
+    );
+    let body = get_text(format!("http://{addr}/metrics")).await;
+    assert!(body.contains("mygauge 42"));
```

Comment:
This doesn't verify that the prefix isn't added.

Reply:
Ah, yeah good point thanks.